fundamentals-review.livemd

```diff
@@ -76,6 +83,14 @@ defmodule Grade do
   "D"
   """
   def letter_grade(integer) do
+    cond do
```

Comment:
cond is a great choice since these are conditions.

Consider moving the comparison like so to make it more readable:

```elixir
72 <= integer and integer <= 85 -> "B"
```